src/style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+  --moyo-blue: #00acdc;
+}
+
+.picture-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.picture-container img {
+  max-width: 100%;
+  max-height: 100%;
+  object-fit: contain;
+}
+
+.product-card p {
+  margin: 0;
+  user-select: none;
+}
+
+.product-card {
+  width: 200px;
+  padding: 30px 14px 15px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.picture-container_product-card {
+  height: 134px;
+  width: 160px;
+  margin: auto;
+}
+
+.product-card__title {
+  display: block;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-top: 45px;
+  margin-bottom: 5px;
+  color: inherit;
+  text-decoration: none;
+}
+
+.product-card__code {
+  padding-bottom: 15px; /* why it dont work with margin property? */
+}
+
+.product-details-row {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.product-stars {
+  display: flex;
+}
+
+.product-stars__star {
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background: url(/images/star.svg) center no-repeat;
+}
+
+.product-stars__star:nth-child(-n+4) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
add rules for -n+1, -n+2 etc so when I change class, count of active stars also changes

Reply:
done